src/Graphics/src/Graphics/Platforms/iOS/PlatformGraphicsView.cs

```diff
@@ -8,9 +8,9 @@ namespace Microsoft.Maui.Graphics.Platform
 	[Register(nameof(PlatformGraphicsView))]
 	public class PlatformGraphicsView : UIView
 	{
-		private WeakReference<IGraphicsRenderer> _renderer;
+		private IGraphicsRenderer _renderer;
```

Comment:
This might still be an issue because PlatformGraphicsView references IGraphicsRenderer which has a reference back to PlatformGraphicsView via the DirectRenderer.GraphicsView property. Setting PlatformGraphicsView.Renderer sets the DirectRenderer.GraphicsView property to the same view, thus a circular reference in all cases.

Reply:
https://github.com/dotnet/maui/pull/19347#discussion_r1423227392
I ran the leak tests locally and they all passed.